src/Tasks/Common/Resources/Strings.resx

```diff
@@ -678,4 +678,8 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1143: Including all content in a single file bundle also includes native libraries. If IncludeAllContentForSelfExtract is true, IncludeNativeLibrariesForSelfExtract must not be false.</value>
     <comment>{StrBegin="NETSDK1143: "}</comment>
   </data>
+  <data name="ILLinkFailed" xml:space="preserve">
+    <value>NETSDK1144: Optimizing assemblies for size failed. You can either fix the errors, or set the PublishTrimmed property to false.</value>
```

Comment:
I don't love the comment "fix the errors" because there's almost certainly going to be a case where the linker crashes or circumstances make it impossible to fix the errors. It's probably fine to just say, "Optimizing assemblies for size failed. Optimization can be disabled by setting the PublishTrimmed property to false."